test/test-platform-output.c

```diff
@@ -152,15 +154,25 @@ TEST_IMPL(platform_output) {
   uv_free_interface_addresses(interfaces, count);
 
   err = uv_os_get_passwd(&pwd);
-  ASSERT(err == 0);
+  ASSERT_EQ(err, 0);
+
+  err = uv_os_get_group(&grp, pwd.gid);
+  ASSERT_EQ(err, 0);
+  ASSERT_EQ(pwd.gid, grp.gid);
 
   printf("uv_os_get_passwd:\n");
   printf("  euid: %ld\n", pwd.uid);
-  printf("  gid: %ld\n", pwd.gid);
+  printf("  gid: %ld (%s)\n", pwd.gid, grp.groupname);
+  printf("    members: [");
+  for (member = grp.members; *member != NULL; member++) {
+    printf("%s", *member);
```

Comment:
```suggestion
    printf(" %s", *member);
```
Otherwise theyallgetsmushedtogether.